systemtest/src/test/java/io/strimzi/systemtest/CustomResourceStatusST.java

```diff
@@ -307,6 +321,16 @@ void waitForKafkaConnectS2IStatus(String status) {
         LOGGER.info("Kafka ConnectS2I cluster is in desired state: {}", status);
     }
 
+    void waitForKafkaTopic(String status, String topicName) {
```

Comment:
Don't we have these types of methods in `StUtils` or it makes sense to have here ? Why do you decided put it here ?

Reply:
After discussion, we will address this as part of future refactor in October.